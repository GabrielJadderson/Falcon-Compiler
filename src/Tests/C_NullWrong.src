# Typecheck of use of null, that the type of null is not compatible with
# the integer type
var a: array of int;
var b: int;

allocate a of length 10;
b = null;
write 42;
