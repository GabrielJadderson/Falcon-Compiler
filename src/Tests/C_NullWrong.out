Compiler should catch the error
